### PR TITLE
No sql databases

### PR DESCRIPTION
**Pull request by VitalySokolov**

### CODE REVIEW

File: src/models/populate-mongodb.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    await addUser(user);
+  }
+};
+/* eslint-enable no-await-in-loop */
+/* eslint-enable no-restricted-syntax */
+
+populateCities()
+  .then(() => populateProducts())
+  .then(() => populateUsers())
+  .then(() => process.exit(1));
```
Comment (pavel-ulasavets) [inline src/models/populate-mongodb.js]: Why do you exit with code=1 ? code=1 is considered to be a failure according to (https://nodejs.org/dist/latest-v11.x/docs/api/process.html#process_process_exit_code) but according to the context provided in the code base it looks more like normal situation

File: src/routes/cities.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
+const express = require('express');
+const { City, validate } = require('../models/city');
```
Comment (pavel-ulasavets) [inline src/routes/cities.js]: There is no "validate" function exported from ../models/city (rather validateCity). wrong name of the function here, clearly, leads to broken post/put requests

**Review by pavel-ulasavets — CHANGES_REQUESTED** [review]
In general code looks good but there are a couple of functional gaps:

- point #4 (one with a web page) is not implemented
- cities PUT and POST end points crash on incoming requests

Please, fix them

File: src/models/populate-mongodb.js (hunk 1 above)
Comment (VitalySokolov) [reply src/models/populate-mongodb.js]: Thank you. It is just a typo.

File: src/routes/cities.js (hunk 2 above)
Comment (VitalySokolov) [reply src/routes/cities.js]: My bad. Changed it after testing all cities functionality.
Fixed.

**Review by pavel-ulasavets — APPROVED** [review]

**Pull request merged**